README.md

```diff
@@ -74,3 +74,13 @@ digital-repository/$ rspec spec/
 ```
 digital-repository/$ cap <stage> deploy
 ```
+
+##### Digital Hub Export
+* Export repo to JSON formatted for InvenioRDM
+```
+./bin/rake repo_export
+```
+* Same as above, but with extra debugging information
+```
+./bin/rake verbose debug repo_export
```

Comment:
It's weird, but chaining the rake tasks persists the logger and it's config.

Reply:
I see, I was wondering why both "verbose" and "debug" were needed.